spec/models/categories_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  context "successfully creates a value in the category table" do 
+
+  	category = FactoryBot.create(:category)
+  	it "should write a value to the category table" do 
+  		expect(category.save).to eq(true)
+		end
+	end
+
+	context "category model has validations" do
+		let!(:category1) { Category.new(name: 'hello') }
+		let!(:category2) { Category.new(name: 'hello') }
+		let!(:category3) { Category.new(name: 'goodbye') }
+
+		it "should not store a duplicate name in the category table" do
+			expect(category1.save).to eq(true)
+			expect(category2.save).to eq(false)
+		end
+
+		it "should store a unique name" do
+			expect(category3.save).to eq(true)
+		end
+	end
+
+end
```

Comment:
We should have ActiveRecord `shoulda` validations in here. Refer to the Week 5 reading.

Reply:
talked to Nick, we'll be sure to use Shoulda moving forward